appinventor/components/src/com/google/appinventor/components/common/YaVersion.java

```diff
@@ -419,8 +419,10 @@ private YaVersion() {
   // For YOUNG_ANDROID_VERSION 166:
   // - MAP_COMPONENT_VERSION was incremented to 3
   // - FEATURE_COLLECTION_COMPONENT_VERSION was incremented to 2
+  // For YOUNG_ANDROID_VERSION 167:
+  // - LOCATIONSENSOR_COMPONENT_VERSION was incremented to 4
 
-  public static final int YOUNG_ANDROID_VERSION = 166;
+  public static final int YOUNG_ANDROID_VERSION = 167;
```

Comment:
`YOUNG_ANDROID_VERSION` should be only updated when we officially publish to a server, similarly to how we handle the change to the Companion version.

Reply:
In fact, this should be a part of a comapnion  version upgrade. If I keep this, the property update would not be triggered. 
And this is why i had to push it to branch ucr instead of master, more upgraded can be still added into this version 167 before ucr is merged.

Reply:
No, this version is only updated when we push a new version to an official server and it is done by MIT team. We change this version after there are multiple new features. For instance, you can take a look into #1070 pull request, which is ready to be pushed for Gerrit, but does not update Young Android version.

Reply:
I know what you mean, currently I just need to make sure that this upgrade is triggered.

@ewpatton Should I draw back the change on `YOUNG_ANDROID_VERSION` here?

Reply:
@ColinTree your upgrade will be still triggered, because you edited LocationSensor upgrader.

Reply:
Nope it would not. I tried before edit this.

Reply:
Interesting enough, my click event for image component PR upgrades without Young Android version update.

Reply:
Cuz you dont need to trigger any property upgrader, there are no changes between version 3 & 4, but this PR do.

Reply:
...and as I said before YOUNG_ANDROID_VERSION label is being updated by MIT team, so both yours and mine PRs will work even without us changing the integer number.

Reply:
You are both "correct." Yes, YOUNG_ANDROID_VERSION needs to be incremented to make this change work. But in order to make it easier for us to manage multiple changes to the language at the same time, we are asking that people not update the YOUNG_ANDROID_VERSION because it causes conflicts when merging. MIT will update the YOUNG_ANDROID_VERSION when we perform a release.